### PR TITLE
First draft for the GRASS 8.3.0 news

### PR DESCRIPTION
**Pull request by veroandreo**
Some links need to be added, i.e., windows binaries, and a screenshot is also missing. 

I didn't include anything about the community meeting as I thought that maybe creating a news with a wrap up would be better, but please feel free to edit directly if a note on that should be included.

### CODE REVIEW

File: content/news/2023_06_11_GRASS_GIS_8_3_RC1_released.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+title: "First release candidate for GRASS GIS 8.3 is out"
+date: 2023-06-11T12:42:00-03:00
+layout: "news"
+author: GRASS Development Team
+---
+
+### What's new in a nutshell
+
+The GRASS GIS 8.3.0RC1 release provides more than 360 improvements 
+and fixes with respect to the 8.2 branch. This new minor release
```
Comment (neteler) [inline content/news/2023_06_11_GRASS_GIS_8_3_RC1_released.md:11]:
```suggestion
and fixes compared to the 8.2 branch. This new minor release
```

File: content/news/2023_06_11_GRASS_GIS_8_3_RC1_released.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+brings in many fixes and improvements in GRASS GIS modules and the
+graphical user interface (GUI) which now has the 
+**single window layout by default**. 
+Some of the most relevant changes include: support for parallelization
+in three raster modules, new options added to different temporal modules,
```
Comment (neteler) [inline content/news/2023_06_11_GRASS_GIS_8_3_RC1_released.md:16]:
```suggestion
in three raster modules, new options added to several temporal modules, 
```

File: content/news/2023_06_11_GRASS_GIS_8_3_RC1_released.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+The GRASS GIS 8.3.0RC1 release provides more than 360 improvements 
+and fixes with respect to the 8.2 branch. This new minor release
+brings in many fixes and improvements in GRASS GIS modules and the
+graphical user interface (GUI) which now has the 
+**single window layout by default**. 
+Some of the most relevant changes include: support for parallelization
+in three raster modules, new options added to different temporal modules, 
+and substantial clean-up of g.extension, the module alowing to install 
+addons. The GUI also received a lot of attention with many fixes and
```
Comment (neteler) [inline content/news/2023_06_11_GRASS_GIS_8_3_RC1_released.md:18]:
```suggestion
and substantial clean-up of `g.extension`, the module that allows the installation
of add-ons. The GUI also received a lot of attention with many fixes and 
```

File: content/news/2023_06_11_GRASS_GIS_8_3_RC1_released.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+graphical user interface (GUI) which now has the 
+**single window layout by default**. 
+Some of the most relevant changes include: support for parallelization
+in three raster modules, new options added to different temporal modules, 
+and substantial clean-up of g.extension, the module alowing to install 
+addons. The GUI also received a lot of attention with many fixes and 
+items being re-organized.
+Furthermore, we have adopted Clang-Format and indented most of C code 
+accordingly. A lot of effort has been put in cleaning up of C/C++ code 
+to fix almost all compiler warnings.
```
Comment (neteler) [inline content/news/2023_06_11_GRASS_GIS_8_3_RC1_released.md:22]:
```suggestion
items reorganised.
We have also adopted the Clang format and indented most of the C code
accordingly. A lot of effort was put into cleaning up the C/C++ code
to fix almost all compiler warnings.
```

File: content/news/2023_06_11_GRASS_GIS_8_3_RC1_released.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+
+Tanslations were moved from Transifex to 
+[Weblate](https://weblate.osgeo.org/projects/grass-gis/), that 
+automatically creates pull requests with the translated chunks. 
+We'd like to thank translators of all languages for your long-term 
+support!!
```
Comment (neteler) [inline content/news/2023_06_11_GRASS_GIS_8_3_RC1_released.md:28]:
```suggestion
Translations have been moved from Transifex to
[Weblate](https://weblate.osgeo.org/projects/grass-gis/), which
automatically creates pull requests with the translated chunks.
We'd like to thank the translators of all languages for their
long term support!
```

File: content/news/2023_06_11_GRASS_GIS_8_3_RC1_released.md [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+to fix almost all compiler warnings.
+
+Tanslations were moved from Transifex to 
+[Weblate](https://weblate.osgeo.org/projects/grass-gis/), that 
+automatically creates pull requests with the translated chunks. 
+We'd like to thank translators of all languages for your long-term 
+support!! 
+
+Also, docker images have been updated and moved from mundialis 
+organization to <https://hub.docker.com/r/osgeo/grass-gis/>.
```
Comment (neteler) [inline content/news/2023_06_11_GRASS_GIS_8_3_RC1_released.md:31]:
```suggestion
Also, docker images have been updated and moved from the mundialis 
to the OSGeo organization at  <https://hub.docker.com/r/osgeo/grass-gis/>.
```

File: content/news/2023_06_11_GRASS_GIS_8_3_RC1_released.md [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+
+We have carried out quite some work in the GitHub Actions: 
+we added support for "pre-commit" in order to reduce unnecessary 
+runs of the automated checks, there were notable improvements in 
+the code checking section and we have activated renovatebot 
+to automatically maintain GitHub Actions.
+
+Last but not least, we have significantly improved the automated
+release creation to reduce maintainers burden and we have 
+gained nine new contributors! Welcome!!
```
Comment (neteler) [inline content/news/2023_06_11_GRASS_GIS_8_3_RC1_released.md:41]:
```suggestion
release creation to reduce maintainer workload and we have 
gained nine new contributors! Welcome all!!
```

**Comment by neteler** [comment]
One thing:

https://github.com/OSGeo/grass/blob/main/doc/howto_release.md#update-hugo-web-site-to-show-new-version

_"For a (final) release (**not release candidate**), write announcement and publish it"_

We need to decide if to rewrite this PR to be used for final release or change the `howto_release.md`.

**Comment by veroandreo** [comment]
> One thing:
> 
> https://github.com/OSGeo/grass/blob/main/doc/howto_release.md#update-hugo-web-site-to-show-new-version
> 
> _"For a (final) release (**not release candidate**), write announcement and publish it"_
> 
> We need to decide if to rewrite this PR to be used for final release or change the `howto_release.md`.

IMHO, we edit this PR for the final release as we have been doing in the last couple of years from what I see from the news ;-)

**Comment by landam** [comment]
@veroandreo I thought that we are releasing news only for final releases not RC... Let's update this PR once 8.3.0 will be out.

File: content/news/2023_06_11_GRASS_GIS_8_3_RC1_released.md [hunk 8]
```diff
@@ -0,0 +1,94 @@
+---
+title: "First release candidate for GRASS GIS 8.3 is out"
```
Comment (landam) [inline content/news/2023_06_11_GRASS_GIS_8_3_RC1_released.md:2]:
```suggestion
title: "GRASS GIS 8.3.0 released"
```

File: content/news/2023_06_11_GRASS_GIS_8_3_RC1_released.md [hunk 9]
```diff
@@ -0,0 +1,94 @@
...
+date: 2023-06-11T12:42:00-03:00
+layout: "news"
+author: GRASS Development Team
+---
+
+### What's new in a nutshell
+
+The GRASS GIS 8.3.0RC1 release provides more than 360 improvements
```
Comment (landam) [inline content/news/2023_06_11_GRASS_GIS_8_3_RC1_released.md:10]:
```suggestion
The GRASS GIS 8.3.0 release provides more than 360 improvements 
```

File: content/news/2023_06_11_GRASS_GIS_8_3_RC1_released.md [hunk 10]
```diff
@@ -0,0 +1,94 @@
...
+   alt="GRASS GIS 8.3"
+   title="GRASS GIS 8.3"
+   width="60%" style="float:right;padding-left:15px;padding-top:20px">
+</a>
+
+### Full list of changes and contributors
+
+For all 360+ changes, see our detailed announcement with the full list of
+features and bugs fixed at
+[GitHub / Releases / 8.3.0RC1](https://github.com/OSGeo/grass/releases/tag/8.3.0RC1).
```
Comment (landam) [inline content/news/2023_06_11_GRASS_GIS_8_3_RC1_released.md:54]:
```suggestion
[GitHub / Releases / 8.3.0](https://github.com/OSGeo/grass/releases/tag/8.3.0).
```

File: content/news/2023_06_11_GRASS_GIS_8_3_RC1_released.md [hunk 11]
```diff
@@ -0,0 +1,94 @@
...
+
+Thank you all contributors!!
+
+## Download and test!
+
+#### Binaries/Installers download
+
+- Windows
+  - [64bit standalone installer]()
```
Comment (landam) [inline content/news/2023_06_11_GRASS_GIS_8_3_RC1_released.md:63]:
```suggestion
  - [64bit standalone installer](https://grass.osgeo.org/grass83/binary/mswindows/native/WinGRASS-8.3.0-1-Setup.exe)
```

File: content/news/2023_06_11_GRASS_GIS_8_3_RC1_released.md [hunk 12]
```diff
@@ -0,0 +1,94 @@
...
+- macOS
+  - [Official Application Bundle](http://grassmac.wikidot.com/downloads)
+- Linux
+  - [Debian](https://tracker.debian.org/pkg/grass)
```
Comment (landam) [inline content/news/2023_06_11_GRASS_GIS_8_3_RC1_released.md:67]:
```suggestion
  - [Debian](https://tracker.debian.org/pkg/grass)
  - [Ubuntu](https://launchpad.net/~ubuntugis/+archive/ubuntu/ubuntugis-unstable)
```

File: content/news/2023_06_11_GRASS_GIS_8_3_RC1_released.md [hunk 13]
```diff
@@ -0,0 +1,94 @@
...
+  - [Fedora/Centos/EPEL](https://src.fedoraproject.org/rpms/grass)
+  - [Gentoo](https://packages.gentoo.org/packages/sci-geosciences/grass)
+
+Further binary packages for other platforms and distributions will follow shortly,
+please check at [software downloads](/download/software/index.html).
+
+#### Source code download
+
+- <https://grass.osgeo.org/grass83/source/>
+- <https://grass.osgeo.org/grass83/source/grass-8.3.0RC1.tar.gz>
```
Comment (landam) [inline content/news/2023_06_11_GRASS_GIS_8_3_RC1_released.md:77]:
```suggestion
- <https://grass.osgeo.org/grass83/source/grass-8.3.0.tar.gz>
```

File: content/news/2023_06_11_GRASS_GIS_8_3_RC1_released.md [hunk 14]
```diff
@@ -0,0 +1,94 @@
...
+- [From GitHub](https://github.com/OSGeo/grass/releases/tag/8.3.0RC1)
```
Comment (landam) [inline content/news/2023_06_11_GRASS_GIS_8_3_RC1_released.md:78]:
```suggestion
- [From GitHub](https://github.com/OSGeo/grass/releases/tag/8.3.0)
```

**Review by landam — DISMISSED** [review]
Please rename to `2023_06_24_GRASS_GIS_8_3_0_released.md`

File: content/news/2023_06_11_GRASS_GIS_8_3_RC1_released.md [hunk 15]
```diff
@@ -0,0 +1,94 @@
+---
+title: "First release candidate for GRASS GIS 8.3 is out"
+date: 2023-06-11T12:42:00-03:00
```
Comment (neteler) [inline content/news/2023_06_11_GRASS_GIS_8_3_RC1_released.md:3]:
```suggestion
date: 2023-06-24T12:42:00-03:00
```

**Comment by neteler** [comment]
> Please rename to `2023_06_24_GRASS_GIS_8_3_0_released.md`

Done

**Review by neteler — APPROVED** [review]

**Comment by veroandreo** [comment]
Screenshot added, merging

image

**Pull request merged**